tests/test_code_gen.py

```diff
@@ -442,6 +445,107 @@ def test_fstrings(self):
         """
         self.assertSrcRoundtripsGtVer(source, (3, 6))
 
+    @unittest.skipUnless(sys.version_info <= (3, 3),
+                         "ast.Name used for True, False, None until Python 3.4")
+    def test_deprecated_constants_as_name(self):
+        self.assertAstEqualsSource(
+            ast.Assign(targets=[ast.Name(id='spam')], value=ast.Name(id='True')),
+            "spam = True")
+
+        self.assertAstEqualsSource(
+            ast.Assign(targets=[ast.Name(id='spam')], value=ast.Name(id='False')),
+            "spam = False")
+
+        self.assertAstEqualsSource(
+            ast.Assign(targets=[ast.Name(id='spam')], value=ast.Name(id='None')),
+            "spam = None")
+
+    @unittest.skipUnless((3, 4) <= sys.version_info <= (3, 8),
+                         "ast.NameConstant introduced in Python 3.4, deprecated in 3.8")
+    def test_deprecated_name_constants(self):
+        self.assertAstEqualsSource(
+            ast.Assign(targets=[ast.Name(id='spam')], value=ast.NameConstant(value=True)),
+            "spam = True")
+
+        self.assertAstEqualsSource(
+            ast.Assign(targets=[ast.Name(id='spam')], value=ast.NameConstant(value=False)),
+            "spam = False")
+
+        self.assertAstEqualsSource(
+            ast.Assign(targets=[ast.Name(id='spam')], value=ast.NameConstant(value=None)),
+            "spam = None")
+
+    @unittest.skipIf(sys.version_info >= (3, 8),
```

Comment:
I think the skip should be removed. These tests work in 3.8 and will continue to work in several future versions.